src/dlls/mscoree/mscoree.cpp

```diff
@@ -1137,13 +1141,22 @@ STDAPI LoadLibraryShimInternal(LPCWSTR szDllName, LPCWSTR szVersion, LPVOID pvRe
     
```

Comment:
(You can delete the FEATURE_CORECLR specific path in the implementation of the method while you are on it.)